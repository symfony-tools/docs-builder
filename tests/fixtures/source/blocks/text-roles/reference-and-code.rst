Next, create the template used to render the field in the ``index`` and ``detail``
`CRUD pages`_.

More about CRUD pages.

.. _`CRUD pages`: https://symfony.com/doc/current/bundles/EasyAdminBundle/crud.html
