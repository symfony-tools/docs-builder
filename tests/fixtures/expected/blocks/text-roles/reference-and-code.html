<p>Next, create the template used to render the field in the <code translate="no" class="notranslate">index</code> and <code translate="no" class="notranslate">detail</code>
<a href="https://symfony.com/doc/current/bundles/EasyAdminBundle/crud.html" class="reference internal">CRUD pages</a>.</p>
<p>More about CRUD pages.</p>
